felles/abac/src/main/java/no/nav/vedtak/sikkerhet/pdp2/PdpConsumerImpl.java

```diff
@@ -0,0 +1,97 @@
+package no.nav.vedtak.sikkerhet.pdp2;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.time.Duration;
+import java.util.Base64;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectReader;
+
+import no.nav.foreldrepenger.konfig.KonfigVerdi;
+import no.nav.vedtak.exception.TekniskException;
+import no.nav.vedtak.log.mdc.MDCOperations;
+import no.nav.vedtak.mapper.json.DefaultJsonMapper;
+import no.nav.vedtak.sikkerhet.pdp2.xacml.XacmlRequestBuilder2;
+import no.nav.vedtak.sikkerhet.pdp2.xacml.XacmlResponse;
+
+@ApplicationScoped
+public class PdpConsumerImpl implements Pdp2Consumer {
+
+    private static final String DEFAULT_ABAC_URL = "http://abac-foreldrepenger.teamabac/application/authorize";
+    private static final String PDP_ENDPOINT_URL_KEY = "abac.pdp.endpoint.url";
+    private static final String SYSTEMBRUKER_USERNAME = "systembruker.username";
+    private static final String SYSTEMBRUKER_PASSWORD = "systembruker.password"; // NOSONAR
+    private static final String MEDIA_TYPE = "application/xacml+json";
+    private static final Logger LOG = LoggerFactory.getLogger(PdpConsumerImpl.class);
+
+    private static final HttpClient CLIENT = HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(10)).build();
```

Comment:
trenger CLIENT,READER være static?  bean er applicationscoped, så bør kunne være vanlige felt, kanskje også evt. overstyres for test elns

Reply:
Done